core/client/app/controllers/signin.js

```diff
@@ -3,6 +3,7 @@ import ValidationEngine from 'ghost/mixins/validation-engine';
 
 var SigninController = Ember.Controller.extend(SimpleAuth.AuthenticationControllerMixin, ValidationEngine, {
     authenticator: 'simple-auth-authenticator:oauth2-password-grant',
+    needs: ['forgotten'],
```

Comment:
the `needs` api is going the way of the dodo - mind switching to the new [Ember.inject.controller](http://emberjs.com/api/classes/Ember.inject.html#method_controller)?